### ISSUE TITLE
[Accessibility] WPF imagebutton read 'graphic' by NVDA scan mode

### ISSUE THREAD

**Issue by wangqiaodong** [dotnet/wpf#5233]
* .NET Core Version: 
* Windows version: windows 10 20H2
* Does the bug reproduce also in WPF for .NET Framework 4.8?: Yes
* Is this bug related specifically to tooling in Visual Studio (e.g. XAML Designer, Code editing, etc...)? If yes, please file the issue via the instructions [here](https://docs.microsoft.com/visualstudio/ide/how-to-report-a-problem-with-visual-studio?view=vs-2019).No
* Security issues and bugs should be reported privately, learn more via our [responsible disclosure guidelines](https://github.com/dotnet/wpf/blob/master/README.md#reporting-security-issues-and-security-bugs). No

 
 
 **Problem description:**

Create a WPF window with following code: 
`<StackPanel>
      <Button Width="200">
         <Image Source="..\Resources\minimize.png"
                Height="5"
                Width="10"
                Margin="0,5,0,0"></Image>
      </Button>
</StackPanel>`

then open NVDA（https://www.nvaccess.org/download/） and use scan mode to read the button, you will get following read result:

MainWindow  window
Object nav mode
Web mode
Scan mode
System  collapsed  Space
button
graphic

Normal button is read correctly. 

 **Actual behavior:** 

Read the button need to press keyboard down arrow twice, first time will read "button" and second time will read "graphic"

 **Expected behavior:**
 For image button, we only need one step down arrow to navigate it.

 **Minimal repro:**

**Comment by anjalisheel-wpf** [comment]
@wangqiaodong

This issue is not reproducible. For me it is working fine as expected, I have inserted two image buttons using the code provided by you. 

![image](https://user-images.githubusercontent.com/85492722/135205362-66ff3e90-30e2-4641-b05f-ddf7cb36a70a.png)
